.changes/next-release/bugfix-Lex Models V2-eca422d9.json

```
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "Lex Models V2",
+  "description": "Lex Models V2 service requires Content-Type header to be application/x-amz-json-1.1 other than other normal rest-json services"
```

nit: remove "in addition to" as it replaces `'Content-Type'` header?
```suggestion
  "description": "Lex Models V2 service requires Content-Type header to be application/x-amz-json-1.1"
```